scenarios/kitsune_continuous_flow/src/main.rs

```diff
@@ -0,0 +1,39 @@
+use kitsune_wind_tunnel_runner::prelude::*;
+use std::time::Duration;
+
+fn agent_setup(ctx: &mut AgentContext<KitsuneRunnerContext, KitsuneAgentContext>) -> HookResult {
+    create_chatter(ctx)?;
+    join_chatter(ctx)?;
+    ctx.runner_context().executor().execute_in_place(async {
+        tokio::time::sleep(Duration::from_secs(15)).await;
+        Ok(())
+    })
+}
+
+fn behavior(
+    ctx: &mut AgentContext<KitsuneRunnerContext, KitsuneAgentContext>,
+) -> anyhow::Result<()> {
+    let message = format!(
+        "message_{}",
+        std::time::UNIX_EPOCH
```

Comment:
With a value like this, it's possible multiple agents might produce an op with the same id? Should the agent id be included here?

Reply:
Yes! Agent index should work too and is easier to read.